### PR TITLE
chore: move libp2p and byte utils to a single `utils` package

### PR DESCRIPTION
**Pull request by danisharora099**
ref: https://github.com/waku-org/js-waku/issues/1043

### CODE REVIEW

**Comment by weboko** [comment]
I am wondering why we have that big of increase to running time in the benchmark above.

**Review by weboko — APPROVED** [review]

**Comment by fryorcraken** [comment]
This PR does not resolve https://github.com/waku-org/js-waku/issues/1043

> utils can then have export path dedicated per domain:
>
> - utils/bytes
> - utils/libp2p
> - etc

File: packages/core/src/lib/filter/index.ts [hunk 1]
```diff
@@ -14,13 +14,13 @@ import type {
...
 } from "@waku/interfaces";
+import { WakuMessage as WakuMessageProto } from "@waku/proto";
 import {
   getPeersForProtocol,
   selectConnection,
   selectPeerForProtocol,
   selectRandomPeer,
-} from "@waku/libp2p-utils";
-import { WakuMessage as WakuMessageProto } from "@waku/proto";
+} from "@waku/utils";
```
Comment (fryorcraken) [inline packages/core/src/lib/filter/index.ts:23]: `from @waku/utils/libp2p` is the expectation from the original issue.

Comment (fryorcraken) [reply packages/core/src/lib/filter/index.ts:23]: Cc @danisharora099 this PR does not implement the behaviour described in the issue it closes.

**Pull request merged**